dedicated-examples/extended-error-tracking/README.MD

```diff
@@ -0,0 +1,55 @@
+# Extended Client-Side Error Tracking
+
+When a JavaScript error occurs on a tested web page, TestCafe stops test execution and posts an error message. Third-party code execution (for example, controls, logging, and analytics services) can cause unexpected errors that are not related to the tested app. You can filter these errors so that they are not included in the test.
```

Comment:
```suggestion
When a JavaScript error occurs on a tested web page, TestCafe stops test execution and posts an error message. Third-party code execution (for example, controls, logging, and analytics services) can cause unexpected errors that are not related to the tested app. You can filter these errors, so that they are not included in the test.
```